CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# matrix-sdk-crypto-wasm unreleased
+
+-   Functions/methods that take a JavaScript `Array` as argument now delete the
+    array's items once their return, for example `Device::requestVerification`,
+    `OlmMachine::updateTrackedUsers`, `OlmMachine::shareRoomKey`,
+    `OlmMachine::queryKeysForUsers`, `OlmMachine:getMissingSessions` and so on.
```

Comment:
I think we need to give a complete list of the affected methods, so that applications can audit their usages.

Reply:
actually, let's not bother with this. People can just go and look at the linked pull request.